### DISCUSSION TITLE
Mount/Trust server certificate inside Wiki-JS container for S3 MinIO

### DISCUSSION THREAD

**Discussion by D1StrX** [requarks/wiki#6338] [Help / Questions]
Environment: Kubernetes
Trying to configure S3 Generic connection towards MinIO (also hosted in Kubernetes), over https on internal k8s dns. **Over http it works.**
- MinIO svc runs on port 443. Thus tried the URI to contain https:// and without. SSL checked, as shown in the screenshot.
![image](https://user-images.githubusercontent.com/40062371/230973501-a24d08a6-a65d-4e24-94af-688a9288e358.png)
- Mounted the server certificate inside the container, verified its there with the correct data. The certificate is a cert given out by the Kubernetes CA. (So it should be automatically be trusted?)

The only error I see on the GUI is 
```
Error Message
unable to verify the first certificate
``` 
Pods logging only show `[MASTER] info: (STORAGE/S3Generic) Initializing...` 

Any clue what is going on or to look for?

**Comment by Vertiwell** [comment]
I also have this issue, even if I overwrite the ca.crt:
  - name: ca-store
    mountPath: /etc/ssl/certs/ca-certificates.crt
    subPath: ca-certificates.crt
 
I get "unable to get local issuer certificate", if I run curl -v https://minio.example.com:9000 it works fine and returns that the cert is trusted.
I have a bunch of other apps that use S3 with no issues.
